src/generator/protocol/operations.ts

```diff
@@ -171,8 +187,11 @@ function formatParamValue(param: Parameter, imports: ImportManager): string {
 
 // use this to generate the code that will help process values returned in response headers
 function formatHeaderResponseValue(header: LanguageHeader, imports: ImportManager, respObj: string): HeaderResponse {
-  if (respObj[respObj.length-1] == '}') {
-    respObj = respObj.substring(0, respObj.length-1);
+  const dateFormat = '2006-01-02';
+  const datetimeFormat = 'time.RFC3339';
+  const datetimeRFC1123Format = 'time.RFC1123';
```

Comment:
Looks like the same constants are defined earlier, you could probably move them to the file-level and share them.